public-api/src/main/java/org/pmiops/workbench/service/BigQueryService.java

```diff
@@ -103,10 +104,23 @@ public QueryJobConfiguration filterBigQueryConfig(QueryJobConfiguration queryJob
     }
 
     public Long getLong(List<FieldValue> row, int index) {
+        return row.get(index).isNull() ? 0L : row.get(index).getLongValue();
+    }
+
+    public Double getDouble(List<FieldValue> row, int index) {
+        return row.get(index).isNull() ? 0.0 : row.get(index).getDoubleValue();
+    }
+
+    public List<String> getList(List<FieldValue> row, int index) {
         if (row.get(index).isNull()) {
             throw new BigQueryException(500, "FieldValue is null at position: " + index);
         }
-        return row.get(index).getLongValue();
+        List<FieldValue> rowValues = row.get(index).getRepeatedValue();
+        List<String> rowStringValues = new ArrayList<>();
+        for(FieldValue rowValue : rowValues) {
+            rowStringValues.add(rowValue.getStringValue());
+        }
+        return rowStringValues;
```

Comment:
You can use a stream for this.
```suggestion
        return row.get(index).getRepeatedValue().stream()
              .map(FieldValue::getStringValue)
              .collect(Collectors.toList())
```

Reply:
Thanks. Changed this.